code/game/machinery/computer/Operating.dm

```diff
@@ -7,18 +7,18 @@
 	icon_keyboard = "med_key"
 	icon_screen = "crew"
 	circuit = /obj/item/circuitboard/operating
-	var/obj/machinery/optable/table = null
-	var/mob/living/carbon/human/victim = null
+	var/obj/machinery/optable/table
 	light_color = LIGHT_COLOR_PURE_BLUE
-	var/verbose = 1 //general speaker toggle
-	var/patientName = null
+	var/verbose = TRUE //general speaker toggle
 	var/oxyAlarm = 30 //oxy damage at which the computer will beep
-	var/choice = 0 //just for going into and out of the options menu
-	var/healthAnnounce = 1 //healther announcer toggle
-	var/crit = 1 //crit beeping toggle
+	var/choice = FALSE //just for going into and out of the options menu
+	var/healthAnnounce = TRUE //healther announcer toggle
+	var/crit = TRUE //crit beeping toggle
 	var/nextTick = OP_COMPUTER_COOLDOWN
 	var/healthAlarm = 50
-	var/oxy = 1 //oxygen beeping toggle
+	var/oxy = TRUE //oxygen beeping toggle
+	var/currentPatient //Who is on the Operating Table connected to the respective Operating Computer?
```

Comment:
Let's specify the variable type as well: 
```suggestion
	var/mob/living/carbon/currentPatient //Who is on the Operating Table connected to the respective Operating Computer?
```

Reply:
Not sure why I didn't do this in the first place! Thank you!